frontend/document/credocument.lua

```diff
@@ -132,9 +132,9 @@ function CreDocument:init()
     self.info.configurable = true
 end
 
-function CreDocument:loadDocument()
+function CreDocument:loadDocument(only_metadata)
```

Comment:
I wonder if it might be clearer as something like `full_document` (default true, false for metadata only)? Then again, this way it's consistent with crengine, except there we didn't have a choice.

Reply:
But loadDocument() is the generic "loading", `only_metadata` is the exception.
`full_document` if absent, considered `true`, will make it less clear I think:
`self._document:loadDocument(self.file, full_document == false)` ?
Or:
```lua
local only_metadata = full_document == false
self._document:loadDocument(self.file, only_metadata)
```